# Goal
string command
float64 pga
float64 pgv
---
# Result
bool success
---
# Feedback
string status
